webpack.config.js

```diff
@@ -3,10 +3,7 @@
 
 const path = require("path");
 const analyzer = require("webpack-bundle-analyzer");
-const FileManagerPlugin = require("filemanager-webpack-plugin-fixed");
 const { CleanWebpackPlugin } = require("clean-webpack-plugin");
-const TerserPlugin = require("terser-webpack-plugin");
-const JavaScriptObfuscator = require("webpack-obfuscator");
 
 const config = {
```

Comment:
What happened to the obfuscation stuff? Did it not work?

Reply:
That's now on PPA, not here.